LNPopupController/LNPopupController/Private/LNPopupController.m

```diff
@@ -447,9 +447,10 @@ - (void)_popupBarPresentationByUserPanGestureHandler:(UIPanGestureRecognizer*)pg
 			BOOL panThreshold = CACurrentMediaTime() - _lastSeenMovement <= LNPopupBarGesturePanThreshold;
 			BOOL heightTreshold = [self _percentFromPopupBar] > LNPopupBarGestureHeightPercentThreshold;
 			BOOL isPanUp = [pgr velocityInView:_containerController.view].y < 0;
-			BOOL hasPassedOffset = [pgr translationInView:_popupBar.superview].y <= LNPopupBarGestureSnapOffset;
+			BOOL isPanDown = [pgr velocityInView:_containerController.view].y > 0;
+			BOOL hasPassedOffset = abs([pgr translationInView:_popupBar.superview].y) <= LNPopupBarGestureSnapOffset;
```

Comment:
This line fixes the fact that `hasPassedOffset` was *always* true in the case of opening, because the translation in the superview is reported in negative pixels. Happy to revert if that was intentional.